tools/serverpod_cli/lib/src/analyzer/models/validation/restrictions/default.dart

```diff
@@ -309,6 +311,27 @@ class DefaultValueRestriction extends ValueRestriction {
     return errors;
   }
 
+  List<SourceSpanSeverityException> _bigIntValueValidation(
+    dynamic value,
+    SourceSpan? span,
+  ) {
+    if (value is Duration) return [];
```

Comment:
```suggestion
    if (value is BigInt) return [];
```

Did not come up in my testing, but maybe this could happen?

Reply:
Fixed type check.